lib/run-compilation.js

```diff
@@ -26,8 +26,8 @@ module.exports = function runCompilation (options, compiler, done) {
         return file.errored;
       });
 
-      if (!options.quiet) {
-        console.log(chalk.yellow(options.formatter(results)));
+      if (options.quiet === false) {
```

Comment:
I removed the default quiet value in this change though. I'll take a closer look at your suggestion later, but could you try installing this branch locally and I would think it will solve the problem.